Header header
string name
geometry_msgs/Pose pose

# Confidence of pose estimate. Between 0 and 1
float64 confidence
